### DISCUSSION TITLE
Draft Preview Mode

### DISCUSSION THREAD

**Discussion by jdc-pub** [quarto-dev/quarto-cli#2768] [Feature Requests]
I don't think this exists already — a `--draft` option for the `quarto preview` command would be an awesome way to preview the website with draft blog posts included!

**Comment by dragonstyle** [comment]
This is a great suggestion - thank you!

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/2781
